Makefile

```diff
@@ -23,13 +23,20 @@ DOCKER_IMAGE_NAME = ${DOCKER_IMAGE}:${DOCKER_IMAGE_TAG}
 
 PORT ?= 6013
 
+VIRTUALENV_ROOT := $(shell [ -z $$VIRTUAL_ENV ] && echo $$(pwd)/venv || echo $$VIRTUAL_ENV)
+
 .PHONY: help
 help:
 	@cat $(MAKEFILE_LIST) | grep -E '^[a-zA-Z_-]+:.*?## .*$$' | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 # ---- LOCAL FUNCTIONS ---- #
 # should only call these from inside docker or this makefile
 
+.PHONY: freeze-requirements
+freeze-requirements: ## create static requirements.txt
+	${VIRTUALENV_ROOT}/bin/pip install --upgrade pip-tools
```

Comment:
Do you think it's worth supporting `venv`? We don't [here](https://github.com/alphagov/notifications-api/blob/master/Makefile#L64), so having sporadic support among our apps is likely low value / confusing.

Also notable we run bare `pip` below, so if we do this here we should make it consistent elsewhere in this file.

Reply:
I'm OK to get rid of it and just use bare `pip` and assume that people are using a correct version provided by pyenv for example. I don't think it is risk free, as people are maybe a bit less likely to be in a virtualenv as they instead just rely on things happening in docker but potential impact is low.

Let's see how we get on!

Reply:
Have pushed up with a rebased commit